.github/workflows/layer-publish.yml

```diff
@@ -80,6 +88,8 @@ jobs:
             aws lambda publish-layer-version \
               --layer-name $LAYER_NAME \
               --license-info "Apache 2.0" \
+              --compatible-architectures $ARCH
+              --compatible-runtimes ${{ inputs.runtimes }}
```

Comment:
what happens if this is empty? won't the command fail here?

Reply:
Oops, good catch. Also missing the trailing `\`.  I'll fix that.